packages/main/src/themes/sap_belize_hcb/Dialog-parameters.css

```diff
@@ -1,7 +1,5 @@
 @import "../base/Dialog-parameters.css";
 
 :root {
-	--_ui5_dialog_resize_handle_color: var(--sapContent_IconColor);
-	--_ui5_dialog_header_focus_width: 0.125rem;
-	--_ui5_dialog_header_focus_offset: -0.25rem;
+
```

Comment:
You can delete these empty files now.

Just make sure that the corresponding `packages/main/src/themes/THEME_NAME/parameters-bundle.css` file contains the import statement from line 1.